README.md

```diff
@@ -24,13 +24,20 @@ The Dapp will facilitate managing permissioning rules and maintaining the list o
 
 At the moment, these instructions won't say anything about linking the Dapp with your Pantheon node. This integration will be documented in the future. The instructions here are targeted to developers working on the Dapp development, not users.
 
-This is the easiest way to get started:
+This is the easiest way to get started for development with the permissioning Dapp:
 
-1. Get rid of your environment variable named `INGRESS_CONTRACT_ADDRESS` - you might need to restart your terminal session after removing it to have your changes applied. If you are using a `.env` file, you can comment the `INGRESS_CONTRACT_ADDRESS` variable.
+1. Get rid of your environment variable named `NODE_INGRESS_CONTRACT_ADDRESS` - you might need to restart your terminal session after removing it to have your changes applied. If you are using a `.env` file, you can comment the `NODE_INGRESS_CONTRACT_ADDRESS` variable.
 1. Start a terminal session and start a truffle Ganache node running `truffle develop`. This will start a Ganache node and create a truffle console session.
 1. In the truffle console, run all migrations from scratch with `migrate --reset`. Keep this terminal session open to maintain your Ganache node running.
 1. In a new terminal session, navigate to the `app/` directory and run `npm install` to install all required dependencies and `npm start` to start the web server that is serving our dapp. You only need to run `npm install` if you are running the app for the first time.
 1. In your browser, connect Metamask to the Ganache network (the default endpoint is `http://127.0.0.1:9545/`)
 1. When you start Ganache, it gives you a list of accounts and private keys. Import the first one in Metamask to impersonate the first admin of the system.
 1. Navigate to `http://localhost:3000` to access the Permissioning Dapp.
 1. All changes made to the smart contracts or to the dapp code are automatically refreshed on the website. There is no need to restart the web server after making changes.
+
+To deploy the permissioning Dapp:
```

Comment:
Do these instructions tell you how to deploy the Dapp, or build it?

Reply:
They are all about building for deployment (the end artifact is a set of static files that should be served by an http server).